tests/browser/registration/test_health_reg.py

```diff
@@ -0,0 +1,182 @@
+    browser.click(who_to_register_selector_invalid)
+    consent_checkbox_name_invalid = "intro-and-consent-0-consent_h_c"
+
+    admin_fields_invalid = [
+        ("id_household-info-0-admin1_h_c", "Abia", "State"),
+        ("id_household-info-0-admin2_h_c", "Aba North", "LGA"),
+        ("id_household-info-0-admin3_h_c", "Ariaria", "Ward"),
+    ]
+
+    for field_id, text_to_select, field_type in admin_fields_invalid:
+        select2_container_selector = f"#{field_id}"
+        try:
+            browser.assert_element_present(select2_container_selector)
+            browser.select2_select(field_id, text_to_select)
+            browser.sleep(1)
+        except Exception as e:
+            is_required = False
+            if field_type == "State":
+                admin2_field_info = next(
+                    (
+                        f
+                        for f in health_registration.flex_form.get_form_class()
+                        .base_fields["household-info"]
+                        .form.flex_form.fields
+                        if f.name == "admin2_h_c"
+                    ),
+                    None,
+                )
+                if admin2_field_info and admin2_field_info.required:
+                    is_required = True
+            elif field_type == "LGA":
+                admin3_field_info = next(
+                    (
+                        f
+                        for f in health_registration.flex_form.get_form_class()
+                        .base_fields["household-info"]
+                        .form.flex_form.fields
+                        if f.name == "admin3_h_c"
+                    ),
+                    None,
+                )
+                if admin3_field_info and admin3_field_info.required:
+                    is_required = True
+
+            if is_required:
+                pytest.fail(f"Required field '{field_id}' ({field_type}) not found or interaction failed: {e}")
+
+    submit_button_selector_invalid = "//input[@type='submit' and @data-msgid='Save']"
+    browser.assert_element_present(submit_button_selector_invalid, by="xpath")
+    browser.click(submit_button_selector_invalid, by="xpath")
+
+    enumerator_error_message = "Enter a valid value"
+    enumerator_error_selector = f"#id_{enumerator_code_name_invalid}_error"
+    try:
+        browser.wait_for_element_visible(enumerator_error_selector, timeout=10)
+        browser.assert_text(enumerator_error_message, selector=enumerator_error_selector)
+    except TimeoutException:
+        pytest.fail(f"Validation error '{enumerator_error_message}' for enumerator code not found or not visible.")
+
+    consent_error_message = "This field is required."
+    consent_error_selector = f"#id_{consent_checkbox_name_invalid}_error"
+    try:
+        browser.wait_for_element_visible(consent_error_selector, timeout=10)
+        browser.assert_text(consent_error_message, selector=consent_error_selector)
+    except TimeoutException:
```

Comment:
this is not needed  browser.assert_text() already raise exception

Reply:
Ok.